data/seeds/009_schedules.js

```diff
@@ -0,0 +1,31 @@
+exports.seed = function (knex) {
+  return knex('schedules').insert([
+    {
+      course_id: 1,
+      start_time: '5:00pm',
+      start_date: '10-10-2010',
+      end_date: '10-10-2010',
+      end_time: '6:00pm',
+      location: 'https://zoom.us/my/john123',
+      instructor_id: 1,
+    },
+    {
+      course_id: 2,
+      start_time: '6:00pm',
+      start_date: '10-10-2010',
```

Comment:
All of these courses being past date may cause some troubles on front end when we start trying to populate the calendar.  Maybe 1 outdated so we can write the logic to understand it's already finished.   2 with end of November end dates so we know they're upcoming?

Reply:
Thinking about it - start date and end dates would both need to be same day later in November, what're your thoughts around this?  @WillisLi